tenable/ot/schema/assets.py

```diff
@@ -144,16 +145,27 @@ class Assets(NodesList):
     nodes: List[Asset]
 
 
+@dataclass
```

Comment:
I think that you can move it to tenable/ot/schema/base.py (the IP calss and the IPList)

Reply:
Done